### PR TITLE
SageMaker Inference Recommender for HuggingFace BERT Sentiment analysis

### PR DESCRIPTION
**Pull request by eitansela**
*Issue #, if available:*

*Description of changes:*
Added a new notebook `huggingface-inference-recommender.ipynb` - with SageMaker Inference Recommender for HuggingFace BERT Sentiment analysis.

*Testing done:*
End to end on SageMaker Studio and SageMaker Notebooks.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb [hunk 1]
```diff
@@ -0,0 +1,989 @@
+{
```
Comment (jkroll-aws) [inline sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: <p>Nit - capitalize <code>Analysis</code>  in title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jkroll-aws) [inline sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: <p>Can you add a little detail around how you are selecting this model version? Are they sorted by ascending latency by default? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Comment (jkroll-aws) [inline sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: <p>Optionally add a Conclusion section for summarizing what was accomplished in this notebook and suggesting next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (eitansela) [reply sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: Done.

Comment (eitansela) [reply sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: I added details about the sorting and how I choose the model with the lowest latency real-time inference.

Comment (eitansela) [reply sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: Fixed.

Comment (eitansela) [reply sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb:1]: Added Conclusion section with Conclusion and next steps.

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**